### PR TITLE
Add Extreme Industrial Greenhouse

### LINKED ISSUE #9449: Make a lategame farm that does not actually plants/harvests
### Your GTNH Discord Username

Boldar#1676

### Your Pack Version

2.1.2.1

### Your Proposal

Inspired from discussion in the Discord, how about implementing a lategame farm that does not actually needs the crops to grow.
You would place it down, plant crops around, supply power and then it would generate the drops according to the number of fully grown crops in range and their speed bonus. Instead of WA'ing the crops, the farm block itself could scale it's output with power. That way more drops would not need more TE's updating, which could help a lot with lag-free ressource generation. This is just a rought idea, would need some balancing of course. I would place this at UV Tier maybe.

### Your Goal

Add an endgame lag free farm.

### Your Vision

It could decrease WA spam for crops and massively increase TPS. A WA on the crop has actually double tps impact - the crop grows faster, and the harvest needs to be done faster as well.

### Final Checklist

- [X] I have searched this issue tracker and there is nothing similar already. Posting on a closed issue saying *I like this feature please reconsider adding it* will prompt us to investigate and reopen it once we confirm your report.
- [X] I believe there is nothing similar in the pack already, or the existing solution isn't good enough.
- [X] I understand this change request may not attract enough attention and thus not be implemented.
- [X] I understand this change request may be rejected due to other community members thinking it's inappropriate.
- [X] I believe this feature would make the pack better.

### PR DESCRIPTION
**Pull request by kuba6000**
Closes https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/9449
image
image
image
image

### CODE REVIEW

File: src/main/java/com/github/bartimaeusnek/bartworks/common/tileentities/multis/GT_TileEntity_ExtremeIndustrialGreenhouse.java [hunk 1]
```diff
@@ -0,0 +1,706 @@
...
+            addInfo("Process time is divided by number of tiers past HV (Minimum 1 sec)").
+            addInfo("Will automatically craft seeds if they are not dropped").
+            addInfo("-------------------- IC2    CROPS --------------------").
+            addInfo("Minimal tier: UV").
+            addInfo("Need UV glass tier").
+            addInfo("Starting with 4 slots").
+            addInfo("Every slot gives 1 crop").
+            addInfo("Every tier past UV, slots are multiplied by 4").
+            addInfo("Process time: 5 sec").
+            addInfo("All crops are accelerated by x32 times").
```
Comment (draknyte1) [inline src/main/java/com/github/bartimaeusnek/bartworks/common/tileentities/multis/GT_TileEntity_ExtremeIndustrialGreenhouse.java:169]: I believe this was removed.

Comment (kuba6000) [reply src/main/java/com/github/bartimaeusnek/bartworks/common/tileentities/multis/GT_TileEntity_ExtremeIndustrialGreenhouse.java:169]: No, It works as described

**Comment by LewisSaber** [comment]
It looks like it can only grow vanilla and Pams seeds in normal mode, is it indeed?

**Comment by kuba6000** [comment]
Should work with all crops that can be placed on the farmland in normal mode

**Comment by LewisSaber** [comment]
i tried ading natura crops to it , witchery and thaumic bases

**Comment by LewisSaber** [comment]
Also growing stonelilys gives no output(i tried red and white(whith 1,1,1 stats and with high stats), bauxia and argentia works fine

**Comment by Dream-Master** [comment]
i merge this now we can test on zeta

**Review by Dream-Master — APPROVED** [review]

**Pull request merged**